src/Ocean/Ocean.jl

```diff
@@ -1,11 +1,14 @@
 module Ocean
 
-export AbstractOceanCoupling, Uncoupled, Coupled
+export AbstractOceanCoupling, Uncoupled, Coupled, AdvectionTerm, NonLinear
 
 abstract type AbstractOceanCoupling end
 struct Uncoupled <: AbstractOceanCoupling end
 struct Coupled <: AbstractOceanCoupling end
 
+abstract type AdvectionTerm end
+struct NonLinear <: AdvectionTerm end
```

Comment:
```suggestion
struct NonLinearAdvectionTerm <: AdvectionTerm end
```

`NonLinear` seem a bit vague by CLIMA standard :)

Reply:
ha, I originally had that but didn't want to type it all out. I can add it back